src/definitions/modules/flyout.js

```diff
@@ -75,8 +75,10 @@ $.fn.flyout = function(parameters) {
         $pusher              = $context.children(selector.pusher),
         $style,
 
+        isFlyoutComponent    = $module.hasClass(namespace),
```

Comment:
The namespace variable is customizable and not necesarily identical to the default css classname. In case one changes that to 
```javascript
settings: {
	namespace: 'flyoutCustom'
}
```
the module is not detected as a flyout component anymore as a `flyoutCustom` css class does not exists.

Reply:
```suggestion
        isFlyoutComponent    = $module.hasClass('flyout'),
```